### PR TITLE
Address hotfix detection

### PR DESCRIPTION
**Pull request by rvermeulen**
## Description

Address incorrect hotfix detection by simply checking the patch version is not 0.

## Change request type

 - [x] Release or process automation (GitHub workflows, internal scripts)
 - [ ] Internal documentation
 - [ ] External documentation
 - [ ] Query files (`.ql`, `.qll`, `.qls` or unit tests)
 - [ ] External scripts (analysis report or other code shipped as part of a release)

## Rules with added or modified queries

 - [x] No rules added
 - [ ] Queries have been added for the following rules:
     - _rule number here_
 - [ ] Queries have been modified for the following rules:
     - _rule number here_

## Release change checklist

A change note ([development_handbook.md#change-notes](https://github.com/github/codeql-coding-standards/blob/main/docs/development_handbook.md#change-notes)) is required for any pull request which modifies:

 - The structure or layout of the release artifacts.
 - The evaluation performance (memory, execution time) of an existing query.
 - The results of an existing query in any circumstance.

If you are only adding new rule queries, a change note is not required.

_**Author:**_ Is a change note required? 
  - [ ] Yes
  - [x] No

🚨🚨🚨
_**Reviewer:**_ Confirm that format of *shared* queries (not the .qll file, the
.ql file that imports it) is valid by running them within VS Code. 
  - [ ] Confirmed

_**Reviewer:**_ Confirm that either a change note is not required or the change note is required and has been added.
  - [x] Confirmed

## Query development review checklist

For PRs that add new queries or modify existing queries, the following checklist should be completed by both the author and reviewer:

### Author

 - [ ] Have all the relevant rule package description files been checked in?
 - [ ] Have you verified that the metadata properties of each new query is set appropriately?
 - [ ] Do all the unit tests contain both "COMPLIANT" and "NON_COMPLIANT" cases?
 - [ ] Are the alert messages properly formatted and consistent with the [style guide](https://github.com/github/codeql-coding-standards/blob/main/docs/development_handbook.md#query-style-guide)?
 - [ ] Have you run the queries on OpenPilot and verified that the performance and results are acceptable?<br />_As a rule of thumb, predicates specific to the query should take no more than 1 minute, and for simple queries be under 10 seconds. If this is not the case, this should be highlighted and agreed in the code review process._
 - [ ] Does the query have an appropriate level of in-query comments/documentation?
 - [ ] Have you considered/identified possible edge cases?
 - [ ] Does the query not reinvent features in the standard library?
 - [ ] Can the query be simplified further (not golfed!)

### Reviewer 

 - [ ] Have all the relevant rule package description files been checked in?
 - [ ] Have you verified that the metadata properties of each new query is set appropriately?
 - [ ] Do all the unit tests contain both "COMPLIANT" and "NON_COMPLIANT" cases?
 - [ ] Are the alert messages properly formatted and consistent with the [style guide](https://github.com/github/codeql-coding-standards/blob/main/docs/development_handbook.md#query-style-guide)?
 - [ ] Have you run the queries on OpenPilot and verified that the performance and results are acceptable?<br />_As a rule of thumb, predicates specific to the query should take no more than 1 minute, and for simple queries be under 10 seconds. If this is not the case, this should be highlighted and agreed in the code review process._
 - [ ] Does the query have an appropriate level of in-query comments/documentation?
 - [ ] Have you considered/identified possible edge cases?
 - [ ] Does the query not reinvent features in the standard library?
 - [ ] Can the query be simplified further (not golfed!)

### CODE REVIEW

File: scripts/release/is-hotfix-release.py [hunk 1]
```diff
@@ -6,45 +6,16 @@
...
-
 def main(args: 'Namespace') -> Literal[0,1]:
     try:
-        merge_base = get_merge_base_of_ref()
-        release_versions = get_release_branches_containing(merge_base)
-        if len(release_versions) == 0:
-            print(f"Info: No release branches found containing merge base {merge_base}", file=stderr)
+        version = Version(args.version)
+        if version.patch > 0:
+            print("true")
```
Comment (knewbury01) [inline scripts/release/is-hotfix-release.py:13]: are these just debugging prints?

File: scripts/release/is-hotfix-release.py [hunk 2]
```diff
@@ -6,45 +6,16 @@
...
+        else:
             print("false")
-            return 0
-
-        for version in release_versions:
```
Comment (knewbury01) [inline scripts/release/is-hotfix-release.py:39]: why did this use to be checking multiple versions? did we change the way this script gets used/is there any other changes anywhere that is using this that need to consider that?

**Review by knewbury01 — COMMENTED** [review]
@rvermeulen just couple of small things, after that would be happy to approve!

File: scripts/release/is-hotfix-release.py (hunk 1 above)
Comment (rvermeulen) [reply scripts/release/is-hotfix-release.py:13]: The script uses stdout to communicate the result. In the action workflow we assign this to an env variable that we can check in a condition.

File: scripts/release/is-hotfix-release.py (hunk 2 above)
Comment (knewbury01) [reply scripts/release/is-hotfix-release.py:39]: discussed in a sync! satisfied with the current logic

**Review by knewbury01 — APPROVED** [review]
lgtm

**Pull request merged**